packages/verification-vue-component/src/style/components/certifaction_verification.scss

```diff
@@ -1,4 +1,6 @@
 @import "../settings/variables";
+@import "buttons";
+@import "digitalTwin/digital_twin";
```

Comment:
Folders should be named with kebab-case and filenames with snake_case, but this file structure looks wrong to me anyway.